mod/wikirate/set/type/cardtype.rb

```diff
@@ -1,3 +1,9 @@
+format do
+  def default_header_args _args
+    voo.variant = :plural
+  end
+end
```

Comment:
long-standing request to make Cardtype pages use plural in their headers.  This was ridiculously easy.